.eslintrc.yml

```diff
@@ -610,6 +611,11 @@ overrides:
       '@typescript-eslint/space-before-function-paren': off
       '@typescript-eslint/space-infix-ops': off
       '@typescript-eslint/type-annotation-spacing': off
+
+      # Validating TS Doc comments. see:
+      # https://tsdoc.org/pages/packages/eslint-plugin-tsdoc/
+      'tsdoc/syntax': error
```

Comment:
Can you please put it before `@typescript-eslint` rules and also make a header similar to this one:
https://github.com/graphql/graphql-js/blob/d82c8e2aa603536c6a305b1c66fb329c4d086302/.eslintrc.yml#L26-L29